<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formadduser</class>
 <widget class="QDialog" name="formadduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>AGGIUNGI FUMETTO</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Nome:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Cognome:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Nome utente:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Data di nascita:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineName"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSN"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineUN"/>
            </item>
            <item>
             <widget class="QLineEdit" name="linePass"/>
            </item>
            <item>
             <widget class="QDateEdit" name="dateofB"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboUType">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>Utente admin</string>
          </property>
          <item>
           <property name="text">
            <string>Utente admin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Utente normale</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3">
         <widget class="QPushButton" name="addUserCancel">
          <property name="text">
           <string>ANNULLA</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="addUser">
          <property name="text">
           <string>AGGIUNGI</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
